CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift

```diff
@@ -157,6 +149,19 @@ final class CameraPreviewViewController: UIViewController {
         setupConstraints()
     }
 
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        if isViewLoaded {
```

Comment:
[Suggestion] Let's extract the frame magic into a separate method and leave a comment there.

Reply:
Done